code/game/turfs/turf.dm

```diff
@@ -49,6 +49,7 @@
 	update_starlight()
 
 /turf/on_update_icon()
+	cut_overlays()
```

Comment:
It feels like this should be something done in individual turf update icon calls? Some turfs don't have overlays, and it's clearer if this is done closer to where the other overlay calls are.

Reply:
For instance, I didn't realize until now that the above add_overlays() call is valid with this here.

Reply:
I'm not sure I follow. The call chain should be preserved where possible and anything that cares about its own overlays will reapply them.

Reply:
My complaint is about clarity, not actual function - I didn't realize that overlays were being cut elsewhere when I first read this PR.